src/System/Brightness.c

```diff
@@ -35,36 +38,61 @@ static io_service_t DisplayIOServicePort(CGDirectDisplayID display)
 
 double GetDisplayBrightness(uint32_t display)
 {
+    CGError        err;
+    int            i;
+    CGDisplayCount    max;
+    CGDirectDisplayID    displayIDs[8];
+    
     io_service_t disp_serv;
-    float brightness;
-    kern_return_t ret;
-
-    disp_serv = DisplayIOServicePort(display);
-    if (0 == disp_serv)
+    float brightness = 0.0;
+    kern_return_t ret = KERN_RETURN_MAX;
+    
+    err = CGGetOnlineDisplayList(8, displayIDs, &max);
+    if(err != kCGErrorSuccess) {
```

Comment:
Please try to maintain the same code style as the rest of the project. In this project left braces (`{`) go to the next line.